sunkit_instruments/goes_xrs/goes_chianti_tem.py

```diff
@@ -0,0 +1,329 @@
+import numpy as np
+import pandas as pd
+from scipy import interpolate
+
+from astropy import units as u
+from astropy.io import fits
+from astropy.time import Time
+from sunpy import timeseries as ts
+from sunpy.data import manager
+from sunpy.time import parse_time
+from sunpy.util.exceptions import warn_user
+
+__all__ = ["calculate_temperature_emiss"]
+
+
+def calculate_temperature_emiss(goes_ts, abundance="coronal"):
```

Comment:
If this is trying to be more explicit, should it be `calculate_temperature_emission_measure`?  Is it very verbose.  On the other hand, tab-complete...?